lib/pages/creature-page.tsx

```diff
@@ -327,7 +330,10 @@ const AutoSizingSvg = React.forwardRef(
         viewBox={`${x} ${y} ${width} ${height}`}
         ref={ref}
       >
-        {props.children}
+        {bgColor && (
+          <rect x={x} y={y} width={width} height={height} fill={bgColor} />
```

Comment:
Just using a rect is recommended by https://stackoverflow.com/a/11293812/2422398, and this seems to work fine during export too.